README.md

```diff
@@ -1,17 +1,189 @@
-# Registers Ruby Client
+# GOV.UK Registers Ruby Client
+
 ## Installation
 In your Gemfile add:
 ```
 gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-ruby-client.git'
 ```
-## Usage
+## Getting started 
+
 ```
 require 'registers_client'
 
 registers_client = RegistersClient::RegistersClientManager.new({ cache_duration: 3600 })
+```
+
+_Note: `cache_duration`  is the amount of time a register is cached in-memory, before being re-downloaded._
+
+### Methods
+
+#### `get_entries`
+
+Get all entries from the register. For example, when changes have been made to the information in the `country` register.
```

Comment:
I think the wording from the API docs is more accurate.
*Get all entries from the register. For example, all updates there have ever been to the `country` register.*